devops/Makefile

```diff
@@ -34,6 +35,17 @@ export FORMIO_ROOT_EMAIL := $(or ${FORMIO_ROOT_EMAIL}, dev_sabc@gov.bc.ca)
 export MONGODB_URI := $(or ${MONGODB_URI}, $$MONGODB_URI)
 export QUEUE_PREFIX := $(or $(QUEUE_PREFIX), {sims-local})
 
+export MAX_WAIT=300 # Maximum wait time in seconds
```

Comment:
is this related to https://github.com/bcgov/SIMS/pull/1932/files#r1191748847

Reply:
No this variable is used to check whether the dbmigration job is completed within the specified time